specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -0,0 +1,3793 @@
+    },
+    "ObjectReplicationPolicyProperties": {
+      "properties": {
+        "policyId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique id for object replication policy."
+        },
+        "enabledTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Indicates when the policy is enabled on the source account."
+        },
+        "sourceAccount": {
+          "type": "string",
+          "description": "Required. Source account name."
+        },
+        "destinationAccount": {
+          "type": "string",
+          "description": "Required. Destination account name."
+        },
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ObjectReplicationPolicyRule"
+          },
+          "description": "The storage account object replication rules."
+        }
+      },
+      "required": [
+        "sourceAccount",
+        "destinationAccount"
+      ],
+      "description": "The Storage Account ObjectReplicationPolicy properties."
+    },
+    "ObjectReplicationPolicyRule": {
+      "properties": {
+        "ruleId": {
+          "type": "string",
+          "description": "Rule Id is auto-generated for each new rule on destination account. It is required for put policy on source account."
+        },
+        "sourceContainer": {
+          "type": "string",
+          "description": "Required. Source container name."
+        },
+        "destinationContainer": {
+          "type": "string",
+          "description": "Required. Destination container name."
+        },
+        "filters": {
+          "$ref": "#/definitions/ObjectReplicationPolicyFilter",
+          "description": "Optional. An object that defines the filter set."
+        }
+      },
+      "required": [
+        "sourceContainer",
+        "destinationContainer"
+      ],
+      "description": "The replication policy rule between two containers."
+    },
+    "ObjectReplicationPolicyFilter": {
+      "properties": {
+        "prefixMatch": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Optional. Filters the results to replicate only blobs whose names begin with the specified prefix."
+        },
+        "minCreationTime": {
+          "type": "string",
+          "description": "Blobs created after the time will be replicated to the destination. It must be in datetime format 'yyyy-MM-ddTHH:mm:ssZ'. Example: 2020-02-19T16:05:00Z"
+        }
+      },
+      "description": "Filters limit replication to a subset of blobs within the storage account. A logical OR is performed on values in the filter. If multiple filters are defined, a logical AND is performed on all filters."
+    },
+    "ErrorResponseBody": {
```

Comment:
It looks the errorresponds structure changed after the code change. 
Have you checked all existing API reference ErrorResponds still can parse the errorresponds correctly after the change?

Reply:
SRP explicitly defines all error response so in practice this is just to conform to OData 4.0 default error response format.